quolance-api/src/main/java/com/quolance/quolance_api/services/entity_services/impl/ProjectServiceImpl.java

```diff
@@ -126,6 +126,14 @@ public void updateProject(Project existingProject, ProjectUpdateDto updateDto) {
                     .build();
         }
 
+        // title and description are blank
+        if (updateDto.getTitle().isBlank() || updateDto.getDescription().isBlank()) {
```

Comment:
this does not work because you should be able to send in the dto just 
```json
{
"title":"New title"
}
```
and no field for description since we do not want to change the description.